### PR TITLE
LPS-189501 - Fix Workflow | Automation | Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
**Ticket:** 
- [LPS-189501](https://liferay.atlassian.net/browse/LPS-189501)

**Test:** 
- MyWorkflowTasks#DueDateMustBeInFuture

**Ticket:**
- [LPS-189497](https://liferay.atlassian.net/browse/LPS-189497)

**Test:**
- MyWorkflowTasks#TableShowsTimeLeft

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/MyWorkflowTasks.macro [hunk 1]
```diff
@@ -40,7 +40,9 @@ definition {
 			locator1 = "MyWorkflowTasks#DIALOG_BOX_DUE_DATE",
 			value1 = ${dueDate});
 
-		Type(
+		var startDateTime = StringUtil.removeSpaces(${variavelDoTime});
+
+		Type.sendKeys(
```
Comment (thaiscabrals) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/MyWorkflowTasks.macro:45]: The behavior has changed. It no longer has a custom html to specify the click. It is necessary to use this function.
All the tests that use this macro have been tested and are in the ticket description

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/MyWorkflowTasks.macro [hunk 2]
```diff
@@ -81,7 +83,9 @@ definition {
 				locator1 = "MyWorkflowTasks#DIALOG_BOX_DUE_DATE",
 				value1 = "${displayDateMonth}/${futureDay}/${displayDateYear}");
 
-			Type(
+			var startDateTime = StringUtil.removeSpaces(${variavelDoTime});
+
+			Type.sendKeys(
```
Comment (thaiscabrals) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/MyWorkflowTasks.macro:88]: The behavior has changed. It no longer has a custom html to specify the click. It is necessary to use this function.
All the tests that use this macro have been tested and are in the ticket description

**Comment by thaiscabrals** [comment]
ci:test:workflow

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/MyWorkflowTasks.macro (hunk 1 above)
Comment (patriciajeanperez) [reply modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/MyWorkflowTasks.macro:45]: Same comment in https://github.com/liferay-bpm-qa/liferay-portal/pull/942/files#r1259125629. Might be best to call dueDateTime?

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**